thali/thalilogger.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var winston = require('winston');
```

Comment:
We should hard code for now a call in our default winston instance to always output everything to console and then file a bug to remove that at some point since it obviously shouldn't go into production.